src/taproot.rs

```diff
@@ -213,112 +193,6 @@ mod test {
     use crate::{compute, traits::Signer, v1, v2};
     use rand_core::OsRng;
 
-    #[test]
```

Comment:
Are these tests no longer valid? Just want to make sure they were deleted intentionally.

Reply:
Yeah, this file is now taproot only, so there's no need to do a non-taproot test.

We originally thought it would be okay to do schnorr without a tweaked key, but after investigating I realized that there was no way to safely do a distributed key without tweaking it.